src/components/Home/HomeResources/HomeResourceContainer/HomeResourceContainer.tsx

```diff
@@ -39,23 +38,30 @@ const HomeRoomItem = (props: RoomProps) => {
             </div>
             <Stack spacing={0.5} className='z-0'>
               <Stack direction='row' spacing={1.5} alignItems='center'>
-                <Circle className={`text-sm`} sx={{ color: `${props.vacancy ? '#76D674' : '#E25454'}` }} />
+                <Circle className={`text-sm`} sx={{ color: `${vacancy ? '#76D674' : '#E25454'}` }} />
                 <Typography gutterBottom variant='h5' component='h2' className='font-Inter'>
-                  {props.roomName}
+                  {props.resource.name}
                 </Typography>
               </Stack>
               <Stack direction='row' spacing={1.5} alignItems='center'>
                 <PersonOutlineOutlined className='text-xl text-bgNoHover' />
                 <Stack spacing={-0.5}>
-                  <Typography className='font-Inter text-bgNoHover'>Access available to:</Typography>
+                  <Typography className='font-Inter text-bgNoHover'>
+                    Access available to: {props.resource.accessRights.map(role => role + ' ')}
+                  </Typography>
                   <Typography className='font-Inter text-bgNoHover'>Insert available people</Typography>
                 </Stack>
               </Stack>
               <Grid container>
-                {DUMMY_SUBJECTS.map(subject => (
-                  <Grid item key={subject.name}>
-                    <div className={`m-1 text-bgWhite text-sm rounded-xl px-4`} style={{ backgroundColor: subject.color }}>
-                      <p className='font-Inter'>{subject.name}</p>
+                {filteredTags.map(tag => (
+                  <Grid item key={tag.id}>
+                    <div
+                      className={`m-1 text-bgWhite text-sm rounded-xl px-4 `}
+                      style={{
+                        backgroundColor: `${tagColor.filter(colorTag => colorTag.id === tag.id).map(colorTag => colorTag.color)}`,
```

Comment:
Don't do this because when you are using real data your TagData will have all the fields you need.

After you fetch your TagData do a data.map and add the colours onto the already existing tags in the backend.

Reply:
Oh got it.